tests/all/integration/api/helpers/test_systemnotifications.py

```diff
@@ -0,0 +1,155 @@
+import unittest
+
+from app import current_app as app
+from tests.all.integration.utils import OpenEventTestCase
+from app.api.helpers.system_notifications import (
+    get_event_exported_actions,
+    get_event_imported_actions,
+    get_monthly_payment_notification_actions,
+    get_monthly_payment_follow_up_notification_actions,
+    get_ticket_purchased_notification_actions,
+    get_ticket_purchased_attendee_notification_actions,
+    get_ticket_purchased_organizer_notification_actions,
+    get_event_published_notification_actions,
+    get_event_role_notification_actions,
+    get_new_session_notification_actions,
+    get_session_schedule_notification_actions,
+    get_next_event_notification_actions,
+    get_session_accept_reject_notification_actions,
+    get_invite_papers_notification_actions
+    )
+from tests.all.integration.setup_database import Setup
+
+
+class TestSystemNotificationHelperValidation(OpenEventTestCase):
+    def setUp(self):
+        self.app = Setup.create_app()
+
+    def test_event_exported(self):
+        """Method to test the actions associated with a notification about an event being successfully exported."""
+
+        with app.test_request_context():
+            request_url = 'https://localhost/some/path/image.png'
+            response = get_event_exported_actions(request_url)
+            self.assertIsInstance(response, list)
```

Comment:
Test the data, checking instance is not enough

Reply:
@iamareebjamal The responses consist of a list of objects, how can I test data for it? The nearest I found is by asserting the number of elements in both lists using `assertCountEqual` but it fails.

Reply:
Paste the response here

Reply:
Test with `assertCountEqual`
```shell
FAIL: test_event_exported (test_systemnotifications.TestSystemNotificationHelperValidation)
Method to test the actions associated with a notification about an event being successfully exported.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/abc/test_systemnotifications.py", line 42, in test_event_exported
    self.assertCountEqual([notification], response)
AssertionError: Element counts were not equal:
First has 1, Second has 0:  <app.models.notification.NotificationAction object at 0x7fea9cce5828>
First has 0, Second has 1:  <app.models.notification.NotificationAction object at 0x7fea9c45cb70>
```

Test with `assertListEqual`
```shell
FAIL: test_event_exported (test_systemnotifications.TestSystemNotificationHelperValidation)
Method to test the actions associated with a notification about an event being successfully exported.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/abc/test_systemnotifications.py", line 42, in test_event_exported
    self.assertListEqual([notification], response)
AssertionError: Lists differ: [<app.models.notification.NotificationAction object at 0x7fcbab70e4a8>] != [<app.models.notification.NotificationAction object at 0x7fcbaa6dbdd8>]

First differing element 0:
<app.models.notification.NotificationAction object at 0x7fcbab70e4a8>
<app.models.notification.NotificationAction object at 0x7fcbaa6dbdd8>

- [<app.models.notification.NotificationAction object at 0x7fcbab70e4a8>]
?                                                                ^^^^^

+ [<app.models.notification.NotificationAction object at 0x7fcbaa6dbdd8>]
?                                                               +++ ^^
```

Test with `assertEqual`
```shell
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/abc/test_systemnotifications.py", line 42, in test_event_exported
    self.assertEqual([notification], response)
AssertionError: Lists differ: [<app.models.notification.NotificationAction object at 0x7f02142b0160>] != [<app.models.notification.NotificationAction object at 0x7f0212ae1128>]

First differing element 0:
<app.models.notification.NotificationAction object at 0x7f02142b0160>
<app.models.notification.NotificationAction object at 0x7f0212ae1128>

- [<app.models.notification.NotificationAction object at 0x7f02142b0160>]
?                                                               - ^^ ^^

+ [<app.models.notification.NotificationAction object at 0x7f0212ae1128>]
?                                                                ^^ ^^^
```

Reply:
Why are these dynamic?

Reply:
On creating an object it is allotted an address in memory which changes everytime the object is created.

Reply:
Why are the list list and object contents dynamic :roll_eyes:

Reply:
> Why are the list list and object contents dynamic :roll_eyes:
Apologies, I think I am not able to understand by the term dynamic and the content referred with it. Can you tell me more about which content is dynamic here.

Reply:
The returned list and its content

Why can't you verify its content to be equal of some predefined value